scripts/validateLinks.js

```diff
@@ -45,7 +45,12 @@ async function load_page_info(page, relative_path) {
         if (match[1].length == 0) {
             if (match[3].length > 0) {
                 // reference to self anchor
-                ret.links.push([relative_path, match[3]]);
+                if (relative_path.endsWith(".html")) {
+                    ret.links.push([relative_path, match[3]]);
```

Comment:
Should this case be turned into an error to ensure we have converted all `.html` links?

Reply:
this shouldn't cause any error since relative paths referred to href attribute values.